docs/reference/nuget-exe-CLI-Reference.md

```diff
@@ -73,3 +73,8 @@ Developers concerned only with consuming packages, for example, need only unders
 
 > [!Note]
 > Command option names are case-insensitive. Options that are deprecated are not included in this reference, such as `NoPrompt` (replaced by `NonInteractive`) and `Verbose` (replaced by `Verbosity`).
+
+## Localization
+
+NuGet.exe's progress, warning and error messages are  translated in the same locales as Visual Studio.
```

Comment:
Should we mention about environmental var which controls it?
Maybe someone using non-english OS still wants to use English error message then we need to inform them how to do.

Reply:
The environment variable is documented in https://learn.microsoft.com/en-us/nuget/reference/cli-reference/cli-ref-environment-variables. 

I don't expect this to be a super common scenario, so I don't think we need to add it.